src/components/QrScanner/QrCamera.tsx

```diff
@@ -53,7 +53,9 @@ export const QrCamera: FunctionComponent<QrCamera> = ({
   onQrData,
   disabled = false
 }) => {
-  const [hasCameraPermission, setHasCameraPermission] = useState();
+  const [hasCameraPermission, setHasCameraPermission] = useState<
+    boolean | undefined
+  >(undefined);
```

Comment:
Do we need to define it as `undefined`? Also, `useState<boolean>` should suffice.

Reply:
Hmm im not sure of the default value for useState boolean, need to check if its undefined!

Reply:
Turns out it returns undefined and is also auto typed if initial value arg is left blank changed the code to reflect the change 👍